scripts/account/docker-clean-up.sh

```diff
@@ -8,8 +8,11 @@ docker compose down frequency redis account-service-api account-service-worker
 echo "Removing specified volumes..."
 docker volume rm gateway_redis_data
 docker volume rm gateway_chainstorage
-docker volume rm gateway_account_node_cache
+docker volume rm gateway_account_api_node_cache
+docker volume rm gateway_account_worker_node_cache
+docker volume prune -f 
```

Comment:
nit: let's use `-f` flag here so the script doesn't complain if any of the volumes don't exist. (I checked and that's all the flag does)